tests/functional/test_s3.py

```diff
@@ -1340,6 +1340,79 @@ def test_500_error_with_non_xml_body(self):
             self.assertEqual(len(http_stubber.requests), 2)
 
 
+class TestS3Parser(BaseS3OperationTest):
+    def get_operations_with_expires(self):
+        s3 = self.session.create_client("s3")
+        service_model = s3.meta.service_model
+        operations_with_expires = []
+        operations = service_model.operation_names
+        for operation in operations:
```

Comment:
I'd rather just pick one where we know it is and use that than iterate through and test each operation.

If something goes wrong here, then we may wind up having this return an empty list, and these couple of tests will not make any assertions and just pass.  Also, what do we gain from testing this multiple times on multiple operations?